core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -571,6 +573,19 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
     return this.platformFeatureService.status
       .ShowFeedbackUpdatesInProfilePicDropdownMenu.isEnabled;
   }
+
+  // Return false for blog page.
+  shouldHideBackButton(): boolean {
+    let currentPath = this.urlService.getPathname();
+    // Find the first matching path from the list of hidden back button paths.
+    let path = this.backButtonHiddenPaths.find(backButtonHiddenPath =>
+      currentPath.includes(backButtonHiddenPath)
+    );
```

Comment:
Can we do a search through the list using the "some" function and return the output boolean at this point?